editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser2.tsx

```diff
@@ -52,6 +52,7 @@ import {
   ConditionalsControlSectionOpenTestId,
   ConditionalsControlSwitchBranchesTestId,
 } from '../sections/layout-section/conditional-section'
+import { getAllUniqueUids } from '../../../core/model/get-unique-ids'
```

Comment:
Not necessary?

Reply:
Ah this is left over from a previous attempt